packages/python-pulp-cli/python-pulp-cli.spec

```diff
@@ -6,7 +6,7 @@
 %global pypi_name pulp-cli
 
 Name:           python-%{pypi_name}
-Version:        0.25.4
+Version:        0.25.6
```

Comment:
The requests upper bound was changed back to <2.32 because there are issues with 2.32, you might want to change that

Reply:
Oh nice. let me fix that